ui/desktop/src/components/context_management/index.ts

```diff
@@ -60,7 +60,7 @@ export function convertApiMessageToFrontendMessage(
     sendToLLM: sendToLLM ?? true,
     id: generateId(),
     role: apiMessage.role as Role,
-    created: apiMessage.created,
+    created: apiMessage.created ?? 0,
```

Comment:
@DOsinga @zanesq not sure about this one - but something had to be here as it seems to now be optional? I don't quite understand the change earlier